chains/request_additional_funding.json

```
@@ -0,0 +1,297 @@
+{
+  "$schema": "./schema.json",
+  "title": "Request additional funding",
+  "cooldown": 999,
+  "trigger": {
+    "target": "money",
+    "comparator": "lte",
+    "value": 10
+  },
+  "events": {
+    "start": {
+      "text": "Aude: “Wake up, the project bank account is low, you need to do something about it“",
+      "choices": [
+        {
+          "text": "“Do you expect me to magically make money appear ?“",
+          "next": [
+            {
+              "event": "talk-with-aude"
+            }
+          ]
+        },
+        {
+          "text": "“You're right, only bad things will happen if I don't solve this.“",
+          "next": [
+            {
+              "event": "aude-tells-you"
+            }
+          ]
+        }
+      ]
+    },
+    "talk-with-aude": {
+      "text": "Aude: “I am designed to help you with these type of issues. Do have I to remember WHO MADE ME ?“",
+      "choices": [
+        {
+          "text": "“I would never dare questioning your judgment.“",
+          "next": [
+            {
+              "event": "aude-tells-you"
+            }
+          ]
+        },
+        {
+          "text": "“Whatever..“",
+          "next": [
+            {
+              "event": "aude-tells-you"
+            }
+          ]
+        }
+      ]
+    },
+    "aude-tells-you": {
+      "text": "Aude: “You probably know what you should do, even if you're not enjoyed to. Have a bit of courage and go on with it.“",
+      "choices": [
+        {
+          "text": "“Oh no, I guess it's that time of the year. I will ask for more funding to my bosses“",
+          "next": [
+            {
+              "event": "request-funding"
+            }
+          ]
+        },
+        {
+          "text": "“There is nothing I have more than speaking to my bosses, I don't need help from nobody, I'll do fine on my own.“",
+          "next": [
+            {
+              "event": "aude-warns-you"
+            }
+          ]
+        }
+      ]
+    },
+    "request-funding": {
+      "text": "You spend way too much time finding out who to contact and to write the actual mail.",
```

```suggestion
      "text": "You spend way more time finding out who to contact than writing the actual spacemail.",
```
ou bien suivant ce que tu voulais dire
```suggestion
      "text": "You spend way too much time finding out who to contact and writing the actual spacemail.",
```